client/master_client_test.go

```diff
@@ -34,13 +34,13 @@ func TestMasterClient(t *testing.T) {
 	require.Len(t, mcli.Endpoints(), 2)
 
 	// dial to an abonrmal server master, will silent error
-	mcli.UpdateClients(ctx, []string{abnormalHost}, "")
+	mcli.UpdateClients(ctx, join, "")
```

Comment:
I think the argument is all server addresses, not the incremental difference . please help check it @amyangfei

Reply:
👍, it is better to use all server addresses here.